lang/php7/Makefile

```diff
@@ -572,6 +581,7 @@ $(eval $(call BuildModule,gettext,Gettext,+PACKAGE_php7-mod-gettext:libintl-full
 $(eval $(call BuildModule,gmp,GMP,+PACKAGE_php7-mod-gmp:libgmp))
 $(eval $(call BuildModule,hash,Hash))
 $(eval $(call BuildModule,iconv,iConv,$(ICONV_DEPENDS)))
+$(eval $(call BuildModule,imap,Imap,+PACKAGE_php7-mod-imap:libopenssl))
```

Comment:
I would prefer: `$(eval $(call BuildModule,imap,IMAP,+PACKAGE_php7-mod-imap:libopenssl))`, i.e. the description in upper case.